src/upstart/ceph-osd.conf

```diff
@@ -22,4 +22,8 @@ instance ${cluster:-ceph}/$id
 export cluster
 export id
 
-exec /usr/bin/ceph-osd --cluster="${cluster:-ceph}" -i "$id" -f --setuser ceph --setgroup ceph
+script
+    . /etc/default/ceph
```

Comment:
We should check for the file before sourcing it. The following seems to work:

`test -f /etc/default/ceph && . /etc/default/ceph`